samples/containers/ninject/Version_6/Sample/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading.Tasks;
+using Ninject;
+using NServiceBus;
+
+public class Program
+{
+    static void Main()
+    {
+        new Program().Start().Wait();
```

Comment:
we use `.GetAwaiter().GetResult();` everywhere else  https://github.com/Particular/docs.particular.net/blob/master/samples/containers/autofac/Version_6/Sample/Program.cs#L10